app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala

```diff
@@ -205,6 +211,43 @@ class RepresenteeController @Inject() (
     }
   }
 
+  def enterDateOfDeath(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withCapacitorOrPersonalRepresentativeAnswers(request) { (representativeType, journey, answers) =>
+      val form = answers.fold(_.dateOfDeath, c => Some(c.dateOfDeath)).fold(dateOfDeathForm)(dateOfDeathForm.fill)
+      val backLink = answers.fold(
+        _ => routes.RepresenteeController.enterName(),
+        _ => routes.RepresenteeController.checkYourAnswers()
+      )
+      Ok(enterDateOfDeathPage(form, backLink, representativeType, journey.isRight))
+    }
+  }
+
+  def enterDateOfDeathSubmit(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withCapacitorOrPersonalRepresentativeAnswers(request) { (representativeType, journey, answers) =>
+      lazy val backLink = answers.fold(
+        _ => routes.RepresenteeController.enterName(),
+        _ => routes.RepresenteeController.checkYourAnswers()
+      )
+      dateOfDeathForm
+        .bindFromRequest()
+        .fold(
+          formWithErrors =>
+            BadRequest(enterDateOfDeathPage(formWithErrors, backLink, representativeType, journey.isRight)),
+          dateOfDeath => {
```

Comment:
if the date of death is equal to the one in session, there's no point doing any updates to the draft return and session